xml/docu_styleguide.language.xml

```diff
@@ -311,13 +313,13 @@
  <sect2 xml:id="sec-prefix">
   <title>Prefixes</title>
   <para>
-   Add a hyphen after the prefix to prefixed words only if:
+   Add a hyphen after the prefix only if:
```

Comment:
Tbh, in this case, neither version makes me happy. :) The old version is imo clearer though.

Reply:
I had to read the old version twice to get the meaning, but I reverted this change.